Package.swift

```diff
@@ -1,10 +1,10 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.10
```

Comment:
I have been upgraded to 5.10, but I can set it to 5.6 for backward compatibility.